package.json

```diff
@@ -102,6 +102,7 @@
     "lodash": "^4.17.21",
     "react-native-paper": "^5.3.1",
     "react-native-vector-icons": "^9.2.0",
+    "react-navigation-header-buttons": "^9.0.1",
```

Comment:
I think we only need this for the storybook and that it isn't used in the library